x-pack/solutions/observability/plugins/apm/test/scout/ui/parallel_tests/settings/custom_links.spec.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect } from '@kbn/scout-oblt';
+import { test } from '../../fixtures';
+
+test.describe('Custom links', { tag: ['@ess', '@svlOblt'] }, () => {
+  test('Viewer should show disabled create button and no edit button', async ({
+    pageObjects: { customLinksPage },
+    browserAuth,
+  }) => {
+    await browserAuth.loginAsViewer();
+    await customLinksPage.goto();
+
+    const createButton = await customLinksPage.getCreateCustomLinkButton();
+    await expect(createButton).toBeDisabled();
+
+    const editButton = await customLinksPage.getEditCustomLinkButton();
+    await expect(editButton).toBeHidden();
+  });
+
+  test('Privileged User should be able to create custom link and delete the created custom link', async ({
+    page,
+    pageObjects: { customLinksPage },
+    browserAuth,
+  }) => {
+    await browserAuth.loginAsPrivilegedUser();
+    await customLinksPage.goto();
+
+    await customLinksPage.clickCreateCustomLink();
+    await expect(page.getByText('Create link')).toBeVisible();
+    await expect(page.getByText('Save')).toBeDisabled();
+
+    // Create a link with unique name to avoid conflicts
+    const uniqueLabel = `test-link-${Date.now()}`;
+    await customLinksPage.fillLabel(uniqueLabel);
+    await customLinksPage.fillUrl('https://example.com');
+
+    await expect(page.getByText('Save')).toBeEnabled();
```

Comment:
it might become flaky if more than one element has "Save" text. I saw it recently in other plugin.

Reply:
Sure, I fixed that, thanks! I was thinking about a different issue I looked at in a different test, sorry 🤦‍♀️